packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -74,6 +74,9 @@ const StyledNavLink = styled.div<{ active?: boolean }>`
     border-bottom: 2px solid
         ${props => (props.active ? props => props.theme.TYPE_DARK_GREY : 'transparent')};
     margin-right: 40px;
+    &:first-child {
+        padding-left: 0;
+    }
```

Comment:
Something is not correct in the navigation bar. See the screenshots

Zeplin - main header and first nav button is more in left than section header
![Screenshot 2021-06-21 at 9 37 32](https://user-images.githubusercontent.com/33235762/122724645-6f489880-d274-11eb-994a-4178cb6f8706.png)

Your branch:
![Screenshot 2021-06-21 at 9 37 38](https://user-images.githubusercontent.com/33235762/122724724-838c9580-d274-11eb-8d18-614b60e45bd6.png)

Reply:
added 3px to header and 5px to button according to zepplin